Cargo.toml

```diff
@@ -26,13 +26,16 @@ name = "tikv-importer"
 [[bench]]
 name = "raftstore"
 harness = false
+path = "benches/raftstore/mod.rs"
 
 [[bench]]
 name = "coprocessor_executors"
 harness = false
+path = "benches/coprocessor_executors/mod.rs"
 
 [[bench]]
 name = "benches"
```

Comment:
Let's rename it to "misc" as well (and we will move benchmarks in misc out in future). Don't forget to modify Makefile as well.